src/hooks/dokumentkrav/useDokumentkravValidation.ts

```diff
@@ -0,0 +1,55 @@
+import { useState } from "react";
+import { DOKUMENTKRAV_SVAR_SEND_NAA } from "../../constants";
+import { IDokumentkrav, IDokumentkravValidationError } from "../../types/documentation.types";
+
+interface IUseDokumentkravValidation {
+  isValid: (list: IDokumentkrav[]) => boolean;
+  getValidationError: (value: IDokumentkrav) => IDokumentkravValidationError | undefined;
+  validationErrors: IDokumentkravValidationError[];
+}
+
+export function useDokumentkravValidation(): IUseDokumentkravValidation {
+  const [validationErrors, setValidationErrors] = useState<IDokumentkravValidationError[]>([]);
+
+  function isValid(dokumentkravList: IDokumentkrav[]) {
+    setValidationErrors([]);
+
+    const unAnswered = dokumentkravList
+      .filter((dokumentkrav) => {
+        return !dokumentkrav.svar;
+      })
+      .map((dokumentkrav) => ({ errorType: "svar", dokumentkrav }));
+
+    const lackingBegrunnelse = dokumentkravList
```

Comment:
Lurer på om `missingBegrunnelse` kunne vært bedre navn? Samme gjelder for `missingFiles`

Reply:
Enig! Jeg fikser 🎉